R/package_installer.R

```diff
@@ -1,7 +1,23 @@
+clean_pkg_str <- function(pkg_str){
```

Comment:
To fix lintr warning:
```suggestion
clean_pkg_str <- function(pkg_str) {
```